docs/how-to/add-the-nginx-route-relation.md

```diff
@@ -42,12 +42,12 @@ charmcraft pack
 ```
 This will build the charm inside an LXC container for you. The output will tell you the location of the built charm. For example, `my-charm_ubuntu-22.04-amd64.charm`.
 
-Now let's add a juju model and deploy our charm.
+Next, add a juju model and deploy the charm.
 ```
 juju add-model ingress-test
 juju deploy ./my-charm_ubuntu-22.04-amd64.charm --resource httpbin-image=kennethreitz/httpbin
 ```
-To inspect the deployment, let's run `juju status`. Once the application reaches a status of `active idle` our application has been deployed. We can visit it in a browser by getting the IP address of the unit and then going to `http://${ip_of_unit}`.
+To inspect the deployment, run `juju status`. Once the application reaches a status of `active idle` our application has been deployed. We can visit it in a browser by getting the IP address of the unit and then going to `http://${ip_of_unit}`.
```

Comment:
```suggestion
To inspect the deployment, run `juju status`. Once the application reaches a status of `active idle` the application has been deployed. Visit it in a browser by getting the IP address of the unit and then going to `http://${ip_of_unit}`.
```
Perhaps an example of the output of `juju status` would be nice to direct the user into where the unit-ip is in the output :D